docs/dev-standards/penetration-testing.md

```diff
@@ -0,0 +1,31 @@
+# Penetration testing
+
+## Introduction
+
+Penetration testing helps identify vulnerabilities that may not be detected by automated tools or internal reviews. It provides an independent assessment of a system’s security posture and supports continuous improvement.
+
+Penetration testing is especially important for systems that handle sensitive data or provide critical services.
+
+## Guidance
+
+Teams **MUST**:
+
+- carry out a penetration test before the first release to production
+- repeat penetration testing at least once a year
+- repeat testing after any significant architectural change
+- track and remediate all findings in a timely manner
+- scope penetration testing based on system risk and data sensitivity
+- ensure tests are conducted by qualified, independent testers
+
```

Comment:
It might also be worth having a "Teams **SHOULD**" sectopm which contains things like:

- Adopt automated penetration testing as part of CI/CD (though this doesnt nigate the requirements of the MUST list).

Reply:
@milkshakeuk i've added a teams should section with the following bullets:

- integrate automated penetration testing tools into CI/CD pipelines to detect common vulnerabilities earlier in the delivery process
- use test environments that closely mirror production to improve the accuracy of findings
- include penetration testing in incident response exercises to validate detection and response capabilities
- review and update test scopes regularly to reflect changes in architecture, threat landscape or data sensitivity
- collaborate with testers during scoping to ensure coverage of high-risk areas and known weak points